src/editors/SelectorPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import PropTypes from 'prop-types';
+import ErrorBoundary from './sharedComponents/ErrorBoundary';
+import { Selector } from './Selector';
+import store from './data/store';
+
+const SelectorPage = ({
+  courseId,
+  lmsEndpointUrl,
+  studioEndpointUrl,
+}) => (
+  <ErrorBoundary>
+    <Provider store={store}>
+      <Selector
+        {...{
+          learningContextId: courseId,
+          lmsEndpointUrl,
+          studioEndpointUrl,
+        }}
```

Comment:
Why not use the props directly here?

Reply:
@giovannicimolin Because I need to initialize the app with [this hook](https://github.com/openedx/frontend-lib-content-components/pull/278/files#diff-9d3cd962fd1e11e4e1f82a58f5609120d0b95ae42a09a2a8935c2ddf09bce531R13). And to use the `useDispatch` I need the component where I am using it to be inside a `Provider`.